docs/api-guide/templates.md

```diff
@@ -0,0 +1,23 @@
+# Templates
+
+Flask API utilizes [blueprints](http://flask.pocoo.org/docs/0.12/blueprints/) for managing browsable pages' template path.
```

Comment:
I think that "0.12" should simply be "latest" so that it will redirect automatically.